app/containers/Popup.tsx

```diff
@@ -28,6 +31,11 @@ const isChromeUrl = (url: string) => {
   return CHROME_URL_REGEX.test(url);
 };
 export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
+  state = {
```

Comment:
move to redux.

Reply:
Since its a `container` component and the `state` defined here is used just to show/hide the `modal` window(not much complication regarding state involved), should we move this to redux ?